pex/pex.py

```diff
@@ -94,6 +94,18 @@ def _extras_paths(cls):
     extras_paths = filter(None, makefile.get('EXTRASPATH', '').split(':'))
     for path in extras_paths:
       yield os.path.join(standard_lib, path)
+    # macOS Sierra support
+    framework = sysconfig.get_config_var("PYTHONFRAMEWORK")
+    if not framework:
+      raise StopIteration()
+    third_party = os.path.join("/Library", framework, sys.version[:3], "site-packages")
+    yield third_party
```

Comment:
It's pretty explicit in the readme about being a thirdparty/extra location.
```sh
jvenus-signal-mbp13:pex jvenus$ cat /Library/Python/2.7/site-packages/README
This directory exists so that 3rd party packages can be installed
here.  Read the source for site.py for more details.
```